app/scripts/modules/core/src/styleguide/src/public/styleguide.html

```diff
@@ -0,0 +1,1139 @@
+<div class="psg-wrapper styleguide">
```

Comment:
Do we need this committed? Seems to be generated at build time, or at the very least, not a part of this PR? I could be wrong...